tests/scripts/thread-cert/command.py

```diff
@@ -2,6 +2,14 @@
 
 import network_layer
 import config
+import mle
+
+def check_address_query(command_msg, destination):
+    """Verify source_node sent a properly formatted Address query request message to destination_node.
```

Comment:
not seeing any `source_node` verification
update the parameter to be consistent with the comments?